packages/quantic/force-app/solutionExamples/main/lwc/exampleSearch/e2e/exampleSearch.e2e.ts

```diff
@@ -0,0 +1,151 @@
+import {testSearch, expect} from './fixture';
+
+let test = testSearch;
+
+test.describe('Example Search Page E2E Tests', () => {
+  test.describe('when loading the search page', () => {
+    test('should render correctly and load results automatically', async ({
+      searchPage,
+    }) => {
+      expect(searchPage.searchbox).toBeVisible();
+      expect(searchPage.facetsManager).toBeVisible();
+      expect(searchPage.facets).toBeVisible();
+      expect(searchPage.summary).toBeVisible();
+      expect(searchPage.sort).toBeVisible();
+      expect(searchPage.tabBar).toBeVisible();
+      expect(searchPage.pager).toBeVisible();
+      expect(searchPage.foldedResultList).toBeVisible();
+      expect(searchPage.recentQueriesList).toBeVisible();
+      expect(searchPage.recentResultsList).toBeVisible();
+
+      const firstResult = searchPage.results.first();
+      expect(firstResult).toBeVisible();
+      expect(await firstResult.textContent()).not.toBeNull();
+
+      const resultsCount = searchPage.results.count();
+      expect(await resultsCount).toBeGreaterThan(0);
```

Comment:
`expect(firstResult).toBeVisible();`
`expect(await resultsCount).toBeGreaterThan(0);`

Pretty sure these two are testing the same thing :) If the first result is visible, the count will be greater than 0